src/frontend/utils/status-label.ts

```diff
@@ -0,0 +1,23 @@
+export class StatusLabel {
```

Comment:
I know you've just moved this to here, just commenting because I missed when this went in: it seems extreme to use a class for this rather than encoding these as 
```
const statusLabels: {
  STATUS_LABEL_WITHIN_THRESHOLD: {
    message: "Within Threshold",
    class: "govuk-tag--green"
  },
  ...
}
```

Reply:
For consistency, this was the same as the `percentage-discrepancy-special-case` used in the invoice banner. I'll change both.